site/src/main/java/uk/nhs/digital/ps/beans/PublicationBase.java

```diff
@@ -115,6 +109,12 @@ public HippoBean getParentDocument() {
         return getPropertyIfPermitted(PropertyKeys.INFORMATION_TYPE);
     }
 
+    public String getFullTaxonomyList() {
+        assertPropertyPermitted(Dataset.PropertyKeys.TAXONOMY);
+
+        return String.join(",", HippoBeanHelper.getFullTaxonomyList(this));
```

Comment:
See similar comment for `Dataset` regarding delegating the 'join' operation to Freemarker's [join](https://freemarker.apache.org/docs/ref_builtins_sequence.html#ref_builtin_join) built-in.

Reply:
Have updated